core/rlwe/scale.go

```diff
@@ -167,7 +167,7 @@ func (s Scale) Min(s1 Scale) (max Scale) {
 // BinarySize returns the serialized size of the object in bytes.
 // Each value is encoded with .Text('x', ceil(ScalePrecision / log2(10))).
 func (s Scale) BinarySize() int {
-	return 21 + (ScalePrecisionLog10+8)<<1 // 21 for JSON formatting and 2*(8 + ScalePrecisionLog10)
+	return 21 + (ScalePrecisionLog10+6)<<1 // 21 for JSON formatting and 2*(8 + ScalePrecisionLog10)
```

Comment:
Comments have not been updated to reflect the changes

Reply:
Fixed.